Makefile

```
@@ -8,8 +12,24 @@ include $(wildcard arch/arm64/boot/dts/*/overlays/Makefile)
 
 DTBO-AMLOGIC	:=	$(addprefix arch/arm64/boot/dts/amlogic/overlays/,$(dtb-amlogic))
 DTBO-ROCKCHIP	:=	$(addprefix arch/arm64/boot/dts/rockchip/overlays/,$(dtb-rockchip))
-DTBO			:=	$(DTBO-AMLOGIC) $(DTBO-ROCKCHIP)
-TMP				:=	$(addsuffix .tmp,$(DTBO))
+DTBO		:=	$(DTBO-AMLOGIC) $(DTBO-ROCKCHIP)
+TMP		:=	$(addsuffix .tmp,$(DTBO))
+
+ARCH ?= arm64
+CROSS_COMPILE ?= aarch64-linux-gnu-
+
+#
+# Build
+#
+.PHONY: build
+build: $(DTBO)
+
+%.dtbo: %.dts
+	cpp -nostdinc -undef -x assembler-with-cpp -E -I "$(KSRC)/include" "$<" "$@.tmp"
+	dtc -q -@ -I dts -O dtb -o "$@" "$@.tmp"
+
+clean_dtbo:
```

```suggestion
clean-dtbo:
```
放到后面clean前面，并注明phony